### PR TITLE
Organize

### PR DESCRIPTION
**Pull request by Dlux804**
File organization is completed.

### CODE REVIEW

**Comment by Dlux804** [comment]
@andreshyer I have fixed the things keeping this PR from passing unit tests.  It is ready for your review.

**Comment by Dlux804** [comment]
@qle2 This branch will merge to master soon (feel free to review and merge it).  It has reorganized many of the files and that affected some of the testing files.  Please make sure to update your testing branch with these updates.

File: README.md [hunk 1]
```diff
@@ -35,6 +35,48 @@ environment on our repo (mlapp.yml).
 ## MLModel Class
 This is the overview of our MLModels Python class functions.  Obviously, it is incomplete just like our code. 
 Update it as you update the code.
-![Alt text](dataFiles/hte-models-overview.png)
+![Alt text](graphics/hte-models-overview.png)
```
Comment (andreshyer) [inline README.md]: addition for graphics is a smart split

File: core/models.py [hunk 2]
```diff
@@ -117,8 +130,9 @@ def store(self):
         # self.graph.savefig(name+'PvA')
 
         # make folders for each run
-        dirsp = 'mkdir ' + name  # str for bash command
-        subprocess.Popen(dirsp.split(), stdout=subprocess.PIPE)  # run bash command
+        # dirsp = 'mkdir ' + name  # str for bash command
+        os.mkdir(name)
+        # subprocess.Popen(dirsp.split(), stdout=subprocess.PIPE)  # run bash command
```
Comment (andreshyer) [inline core/models.py]: Much simplier, and easier to understand

**Pull request merged**